0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -591,9 +591,8 @@ Metadata describing the ledger. This data is returned by the [`getInfo`](#getinf
 |:--|:--|:--|
 | `String` | [prefix](#prefix) | The plugin's ILP address prefix |
 | `Number` | [precision](#precision) | Total number of digits allowed |
-| `Number` | [scale](#scale) | Digits allowed after decimal |
-| `String` | [currencyCode](#currencycode) | ISO three-letter currency code |
-| `String` | [currencySymbol](#currencysymbol) | UTF8 currency symbol |
+| `String` | [currencyCode](#currencyCode) | ISO three-letter currency code |
+| `Number` | [currencyScale](#currencyScale) | Integer, such that `1 <currencyCode>` equals `10^<currencyScale>` ledger units
```

Comment:
I think non-integer scales would be too confusing. But we should definitely support negative scales.